atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/TargetedSweeperTest.java

```diff
@@ -591,6 +593,67 @@ public void testSweepTimestampMetric() {
         TargetedSweepMetricsTest.assertSweepTimestampThoroughEquals(5);
     }
 
+    @Test
+    public void doNotSweepAnythingAfterEntryWithCommitTsAfterSweepTs() {
+        immutableTs = 1000L;
+        enqueueWrite(TABLE_CONS, 900);
+        enqueueWrite(TABLE_CONS, 910);
+        enqueueWriteCommitedAt(TABLE_CONS, 950, 1000);
+        enqueueWrite(TABLE_CONS, 970);
+        enqueueWrite(TABLE_CONS, 920);
```

Comment:
nit: The order here isn't meaningful, right? Might as well order them based on `startTs` to make it clearer.

Reply:
Fair enough